src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -414,9 +415,9 @@ class VAMap extends Component {
         <div>
           {r.type === LocationType.CC_PROVIDER ? (
             <div>
-              <a onClick={linkAction.bind(this, r.id, true)}>
+              <Link to={`/provider/${r.id}`}>
                 <h5>{r.attributes.name}</h5>
-              </a>
+              </Link>
```

Comment:
I think this should be left as an `<a>` tag, but revised a bit to add the `href` values you're calculating. 

```jsx
<a href=[myHref} onClick={linkAction.bind(this)}>
```

I _think_ this will work.

Reply:
I wasn't able to pull the `e.target.href` when I passed in a synthetic event object. I kept the linkAction logic the same as before, and added the href. Works well, going to see if I can figure it out for a bit more simplification.

Reply:
any idea why this doesn't have access to the router context? not super familiar with context api but I didn't think something like the map library we're using could interfere with it.

Reply:
I wasn't able to come up with a good solution other than code in place, and adding an href for keyboard focus.

I'm going to open another issue ticket for continuing to iterate on the map icons. JAWS is not announcing icons because they're empty divs with `tabIndex="0"` attributes. Adding a text string and hiding it with CSS seems to fix that. We could also set focus on the popup using a `ref`, but those are improvements outside this original scope.

Reply:
@rianfowler My first guess is that Mapbox clones the elements and loses something during that, but I'm not sure